prowler/providers/m365/services/defender/defender_chat_report_policy_configured/defender_chat_report_policy_configured.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+from prowler.lib.check.models import Check, CheckReportM365
+from prowler.providers.m365.services.defender.defender_client import defender_client
+
+
+class defender_chat_report_policy_configured(Check):
+    """Check if Defender report submission policy is properly configured for Teams security reporting.
+
+    Attributes:
+        metadata: Metadata associated with the check (inherited from Check).
+    """
+
+    def execute(self) -> List[CheckReportM365]:
+        """Execute the check for Defender report submission policy settings.
+
+        This method checks if Defender report submission policy is properly configured for Teams security reporting.
+
+        Returns:
+            List[CheckReportM365]: A list of reports containing the result of the check.
+        """
+        findings = []
+        report_submission_policy = defender_client.report_submission_policy
+
+        if report_submission_policy:
+            report = CheckReportM365(
+                metadata=self.metadata(),
+                resource=report_submission_policy if report_submission_policy else {},
```

Comment:
```suggestion
                resource=report_submission_policy,
```
It's already inside a `if report_submission_policy:` statement.

Reply:
True! 👍